lib/matplotlib/table.py

```diff
@@ -146,6 +147,67 @@ def set_text_props(self, **kwargs):
         self._text.update(kwargs)
 
 
+class CustomCell(Cell):
+    """
+    A subclass of Cell where the sides may be visibly toggled.
+
+    """
+
+    _edges = 'BRTL'
+    _edge_aliases = {'open':         '',
```

Comment:
@tacaswell afaik spaces are allowed after colons inside of dictionaries.

Nothing against it is mentioned in the PEP 8 style guide: https://www.python.org/dev/peps/pep-0008/

And running pep8 on a minimal example and people on SO seem to confirm: http://stackoverflow.com/questions/13497793/python-alignment-of-assignments-style

I'm also at PyCon right now, I could ask people here too